src/commands/apex/list/log.ts

```diff
@@ -35,61 +36,45 @@ export default class Log extends SfCommand<LogListResult> {
   public async run(): Promise<LogListResult> {
     const { flags } = await this.parse(Log);
 
-    const conn = flags['target-org'].getConnection(flags['api-version']);
-    const logService = new LogService(conn);
-    const logRecords = await logService.getLogRecords();
+    const logService = new LogService(flags['target-org'].getConnection(flags['api-version']));
+    const logRecords = (await logService.getLogRecords()).map(formatStartTime);
 
     if (logRecords.length === 0) {
       this.log(messages.getMessage('noDebugLogsFound'));
       return [];
     }
 
-    logRecords.map((logRecord) => {
```

Comment:
this was doing mutation on the records.  You'll see UT that were relying on that side effect to pass.